tripleblindmarket/private-kit
# [SAF-69] - Autosubscribe to HA on app start

#### Description:
This PR adds a new feature for auto-subscribing users to their local health department after the location permissions have been granted and they have no HAs selected. 


#### How to test:
(Simulator)
- Mock simulator location so that it is within the bounds of at least one known HA from the YAML
- Clean install the app and perform onboarding, granting location permissions
- Upon entering the home screen, the banner should appear at the bottom, saying that you've been subscribed to a HA
- When navigating to `View your Health Departments` screen, the auto-subscribed HA should appear as selected

(Real device)
- Hope that we have a HA in the yaml that has your location within it's bounds -> you should get auto-subscribed after onboarding and get a banner on the bottom of the screen
- Rest is the same as for the simulator
